lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -314,47 +320,36 @@ void FExtModuleOp::build(OpBuilder &builder, OperationState &result,
 // allow this customization.
 static void printFunctionSignature2(OpAsmPrinter &p, Operation *op,
```

Comment:
I think at this point it would make sense to give this function a more meaningful name and remove the TODO